src/Sql/dbo/Stored Procedures/Organization_EnableCollectionEnhancements.sql

```diff
@@ -0,0 +1,153 @@
+                -- Step 1
+                    -- Update existing rows in [dbo].[CollectionGroup]
+                    UPDATE CG
+                    SET
+                        CG.[ReadOnly] = 0,
+                        CG.[HidePasswords] = 0,
+                        CG.[Manage] = 0
+                    FROM [dbo].[CollectionGroup] CG
+                    INNER JOIN [dbo].[Collection] C ON CG.[CollectionId] = C.[Id]
+                    INNER JOIN #TempGroupsAccessAll TG ON CG.[GroupId] = TG.[GroupId]
+                    WHERE C.[OrganizationId] = TG.[OrganizationId];
+
+                    -- Insert new rows into [dbo].[CollectionGroup]
+                    INSERT INTO [dbo].[CollectionGroup] ([CollectionId], [GroupId], [ReadOnly], [HidePasswords], [Manage])
+                    SELECT C.[Id], TG.[GroupId], 0, 0, 0
+                    FROM [dbo].[Collection] C
+                    INNER JOIN #TempGroupsAccessAll TG ON C.[OrganizationId] = TG.[OrganizationId]
+                    LEFT JOIN [dbo].[CollectionGroup] CG ON CG.[CollectionId] = C.[Id] AND CG.[GroupId] = TG.[GroupId]
+                    WHERE CG.[CollectionId] IS NULL;
+
+                    -- Update Group to clear AccessAll flag and update RevisionDate
+                    UPDATE G
+                    SET [AccessAll] = 0, [RevisionDate] = GETUTCDATE()
+                    FROM [dbo].[Group] G
+                    INNER JOIN #TempGroupsAccessAll TG ON G.[Id] = TG.[GroupId];
+
+                -- Step 2
+                    -- Update existing rows in [dbo].[CollectionUser]
+                    UPDATE target
+                    SET
+                        target.[ReadOnly] = 0,
+                        target.[HidePasswords] = 0,
+                        target.[Manage] = 0
+                    FROM [dbo].[CollectionUser] AS target
+                    INNER JOIN [dbo].[Collection] AS C ON target.[CollectionId] = C.[Id]
+                    INNER JOIN #TempUsersAccessAll AS TU ON C.[OrganizationId] = TU.[OrganizationId] AND target.[OrganizationUserId] = TU.[OrganizationUserId];
+
+                    -- Insert new rows into [dbo].[CollectionUser]
+                    INSERT INTO [dbo].[CollectionUser] ([CollectionId], [OrganizationUserId], [ReadOnly], [HidePasswords], [Manage])
+                    SELECT C.[Id] AS [CollectionId], TU.[OrganizationUserId], 0, 0, 0
+                    FROM [dbo].[Collection] C
+                    INNER JOIN #TempUsersAccessAll TU ON C.[OrganizationId] = TU.[OrganizationId]
+                    LEFT JOIN [dbo].[CollectionUser] target
+                        ON target.[CollectionId] = C.[Id] AND target.[OrganizationUserId] = TU.[OrganizationUserId]
+                    WHERE target.[CollectionId] IS NULL;
+
+                    -- Update OrganizationUser to clear AccessAll flag
+                    UPDATE OU
+                    SET [AccessAll] = 0, [RevisionDate] = GETUTCDATE()
+                    FROM [dbo].[OrganizationUser] OU
```

Comment:
If don't need to update organizationuser below maybe do not have to do it here

Reply:
I've updated the other missing `RevisionDate` update